src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/ChartConfiguration.tsx

```diff
@@ -3,15 +3,15 @@ import { useChartBuilderFormsContext } from '@admin/pages/release/datablocks/com
 import { ChartOptions } from '@admin/pages/release/datablocks/components/chart/reducers/chartBuilderReducer';
 import Effect from '@common/components/Effect';
 import FormGroup from '@common/components/form/FormGroup';
-import FormProvider from '@common/components/form/rhf/FormProvider';
-import RHFForm from '@common/components/form/rhf/RHFForm';
-import RHFFormFieldCheckbox from '@common/components/form/rhf/RHFFormFieldCheckbox';
-import RHFFormFieldFileInput from '@common/components/form/rhf/RHFFormFieldFileInput';
-import RHFFormFieldNumberInput from '@common/components/form/rhf/RHFFormFieldNumberInput';
-import RHFFormFieldRadioGroup from '@common/components/form/rhf/RHFFormFieldRadioGroup';
-import RHFFormFieldSelect from '@common/components/form/rhf/RHFFormFieldSelect';
-import RHFFormFieldTextArea from '@common/components/form/rhf/RHFFormFieldTextArea';
-import RHFFormFieldTextInput from '@common/components/form/rhf/RHFFormFieldTextInput';
+import FormProvider from '@common/components/form/FormProvider';
+import Form from '@common/components/form/Form';
+import FormFieldCheckbox from '@common/components/form/FormFieldCheckbox';
```

Comment:
Sorry if I'm missing something obvious, but why do we need to comment these out of the components/form/index file?

Reply:
oops, I just forgot to uncomment them. They're back now.